locations/spiders/migros_ch.py

```diff
@@ -0,0 +1,92 @@
+import json
+
+from scrapy.spiders import SitemapSpider
+
+from locations.hours import DAYS, OpeningHours
+from locations.items import GeojsonPointItem
+
+
+class MigrosCHSpider(SitemapSpider):
+    name = "migros_ch"
+    brands = {
+        "alna": ("Alnatura", "Q876811", {"shop": "supermarket"}),
```

Comment:
I'd prefer it if this used the `Categories` enum, and add the messing ones.

Reply:
Done.